app/src/main/java/com/hedvig/app/feature/marketing/pickmarket/PickMarketScreen.kt

```diff
@@ -0,0 +1,294 @@
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.ModalBottomSheetLayout
+import androidx.compose.material.ModalBottomSheetValue
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Text
+import androidx.compose.material.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.insets.LocalWindowInsets
+import com.hedvig.app.R
+import com.hedvig.app.feature.marketing.PickMarket
+import com.hedvig.app.feature.settings.Language
+import com.hedvig.app.feature.settings.Market
+import com.hedvig.app.ui.compose.composables.buttons.LargeContainedButton
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+import kotlinx.coroutines.launch
+
+enum class PickMarketSheet {
+    MARKET,
+    COUNTRY,
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun PickMarketScreen(
+    onSubmit: () -> Unit,
+    onSelectMarket: (Market) -> Unit,
+    onSelectLanguage: (Language) -> Unit,
+    data: PickMarket
+) {
+    var sheet by rememberSaveable { mutableStateOf<PickMarketSheet?>(null) }
+    val coroutineScope = rememberCoroutineScope()
+    val modalBottomSheetState = rememberModalBottomSheetState(ModalBottomSheetValue.Hidden)
+
+    val insets = LocalWindowInsets.current
+    val navigationBarHeight = with(LocalDensity.current) { insets.navigationBars.bottom.toDp() }
+    ModalBottomSheetLayout(
+        sheetState = modalBottomSheetState,
+        sheetContent = {
+            BackHandler(
+                enabled = modalBottomSheetState.isVisible,
+                onBack = {
+                    coroutineScope.launch { modalBottomSheetState.hide() }
+                },
+            )
+            Spacer(Modifier.height(8.dp))
+            Box(
+                modifier = Modifier
+                    .size(width = 32.dp, height = 4.dp)
+                    .align(Alignment.CenterHorizontally)
+                    .background(
+                        color = colorResource(R.color.color_divider),
+                        shape = RoundedCornerShape(20.dp)
+                    )
```

Comment:
Running this locally I couldn't see this ð Changing it to like Color.Black it was visible. Have you noticed something like that on your local environment too? Might be smth weird on my emulator too somehow

Reply:
Yeah I was running on API 30
It seems to work just fine for dark mode though
<img width="473" alt="image" src="https://user-images.githubusercontent.com/44558292/161580371-ff472c9c-a7e5-4612-9e98-45eab5052336.png">
<img width="473" alt="image" src="https://user-images.githubusercontent.com/44558292/161580940-f12d0f6e-2e78-400e-84c0-1424cfd363b3.png">

Not sure what is going on. Maybe something to do with the theme picking up the wrong thing?
Not sure how it'd ever work on some API and not on some other API after looking at the code. Only thing I can imagine is if it came from a composable function instead with it checking if the system is in dark theme instead if might be a bit more reliable that using `colorResource` with an xml color?

To test this I tried adding this inside `com.hedvig.app.ui.compose.theme.Colors.kt`
```kotlin
val Colors.divider: Color
    @Composable
    @ReadOnlyComposable
    get() = primary.copy(alpha = 0.12f)
```

And using it still doesn't fix it.

But I also see that the radio buttons are also hidden in the light mode. Something must be going wrong in general with light mode and the colors that are picked up.

Reply:
Hmm right I see we're doing this theme overlay with `ThemeOverlay.Hedvig.MarketingActivity` in the `MarketingActivity` so it works, it's just that the primary color is White therefore it's just not showing with the white background. Same for the radio buttons.

Reply:
Damn. Yeah, I see it now. Thanks for the feedback - I'll have to rethink that thing, I guess 😞